docs/docs/legal/rate-limit-policy.md

```diff
@@ -12,18 +12,31 @@ To ensure the availability of our Services and to avoid slow or failed requests
 
 ## How is the rate limit implemented
 
-ZITADEL Clouds rate limit is built around a `IP` oriented model. Please be aware that we also utilize a service for DDoS mitigation.
+ZITADEL Clouds rate limit is built around a `IP` oriented model.
+Please be aware that we also utilize a service for DDoS mitigation.
 So if you simply change your `IP` address and run the same request again and again you might be get blocked at some point.
 
 If you are blocked you will receive a `http status 429`.
 
-:::tip
+:::tip Implement exponential backoff
 You should consider to implement [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff) into your application to prevent a blocking loop.
 :::
 
+:::info Raising limits
+We understand that there are certain scenarios where you must access ZITADEL from one IP Address, for example access behind a corporate proxy or simply many users behind a shared internet connection.
```

Comment:
> or simply many users behind a shared internet connection

You mean NAT?

Reply:
> > or simply many users behind a shared internet connection
> 
> You mean NAT?

Probably... can you make a suggestion? I think mine will not sound as good.

Reply:
```suggestion
We understand that there are certain scenarios where your users access ZITADEL from shared IP Addresses.
For example if you use a corporate proxy or Network Address Translation NAT.
```